src/react-components/src/components/02-molecules/Helpers/HelperErrorText.tsx

```diff
@@ -0,0 +1,31 @@
+/* MT-51 Helper and Error text*/
+import * as React from "react";
+import bem from "../../../utils/bem";
+
+export interface HelperErrorTextProps {
+  id: string;
+  isError: boolean;
+  modifiers?: string[];
+  blockName?: string;
+  baseClass?: string;
+  attributes?: {};
```

Comment:
Is this meant to be any other HTML attribute such as `aria-label`, `data-something`, etc?

If so, you can remove this `attributes` prop. On line 15 you can do
```
const { id, isError, ...rest } = props;
```
And on line 30 you can write
```
return <span id={id} className={bem(blockName, modifiers, baseClass)} {...rest}>{props.children}</span>;
```

`...rest` can be called anything so you can call it `...attributes` if you want to. It just destructures `props` so it holds everything that hasn't been explicitly assigned a variable. It may be easier than having to do 
`<HelperErrorText attributes={ 'aria-label': 'label', ...}>`.